### DISCUSSION TITLE
Set default active tab

### DISCUSSION THREAD

**Discussion by robertosanval** [filamentphp/filament#7869] [Help] [answered]
Hello, I like to set the default tab in a table to the second one.

In the demo https://demo.filamentphp.com/ in `Orders`, there are multiple tabs with the statuses of the orders. I want to set the New tab active by default. 

I can't see a `default()` or `active()` method on `ListRecords\Tab`.

[app/Filament/Resources/Shop/OrderResource/Pages/ListOrders.php](https://github.com/filamentphp/demo/blob/main/app/Filament/Resources/Shop/OrderResource/Pages/ListOrders.php)

```php
    public function getTabs(): array
    {
        return [
            null => ListRecords\Tab::make('All'),
            'new' => ListRecords\Tab::make()->query(fn ($query) => $query->where('status', 'new')),
            'processing' => ListRecords\Tab::make()->query(fn ($query) => $query->where('status', 'processing')),
            'shipped' => ListRecords\Tab::make()->query(fn ($query) => $query->where('status', 'shipped')),
            'delivered' => ListRecords\Tab::make()->query(fn ($query) => $query->where('status', 'delivered')),
            'cancelled' => ListRecords\Tab::make()->query(fn ($query) => $query->where('status', 'cancelled')),
        ];
    }
```

Is there any way to change the URL in the Navigation panel for the Orders Resource to include the `activeTab=new`?

**Comment by Z3d0X** [comment]
You could set the active tab on mount

```php
public function mount(): void
{
    parent::mount();

    if (blank($this->activeTab)) {
        $this->activeTab = 'new';
    }
}
```

**Reply by robertosanval** [reply]
Thanks for your reply.

Where does this code go?

I tried to put it on the OrderResource file, but nothing happens, even if I test with Ray

```php
    public function mount(): void
    {
        ray('TEST');
        parent::mount();
        ray($this->activeTab);

        if (blank($this->activeTab)) {
            $this->activeTab = 'borradores';
        }
    }
```

**Reply by leandrocfe** [reply, marked as answer]
just add in the Pages/ListOrders.php
```php
public ?string $activeTab = 'borradores';
```

**Reply by robertosanval** [reply]
Thanks! It works!
